backend/dataall/core/environment/cdk/environment_stack.py

```diff
@@ -431,6 +432,28 @@ def create_group_environment_role(self, group: EnvironmentGroup, id: str):
             permissions=group_permissions,
         ).generate_policies()
 
+        external_managed_policies = []
+        policy_manager = PolicyManager(
+            environmentUri=self._environment.environmentUri,
+            resource_prefix=self._environment.resourcePrefix,
+            role_name=group.environmentIAMRoleName,
+            account=self._environment.AwsAccountId
+        )
+        for policy in policy_manager.get_all_policies():
```

Comment:
so if share policy does not exist we create it from inline (backwards compatibility) - but we do not attach it to the role (at least not on that specific update)

but if we then update env stack again the managed policies will get attached to the role

OR if we invite a new team to the env their policies will get created during API call and be attached to role on env stack update

…correct?